rqd/rqd/rqutil.py

```diff
@@ -162,6 +162,8 @@ def getHostname():
             return rqd.rqconstants.OVERRIDE_HOSTNAME
         if rqd.rqconstants.RQD_USE_IP_AS_HOSTNAME or rqd.rqconstants.RQD_USE_IPV6_AS_HOSTNAME:
             return getHostIp()
+        if rqd.rqconstants.RQD_USE_FQDN:
+            return socket.gethostbyaddr(socket.gethostname())[0]
```

Comment:
Could you add some unit tests for this function? Ideally for all four of these potential outcomes.

Reply:
Do you think this should go into a new `rqutil_tests.py` file to directly test this function?

Reply:
Yes please.